capstone/src/main/webapp/createStudySet.html

```diff
@@ -1,21 +1,127 @@
 <!DOCTYPE html>
 <html>
   <head>
-    <meta charset="UTF-8">
+    <meta charset="UTF-8" />
     <title>Studentia-Create Study Set</title>
-    <link rel="stylesheet" href="style.css">
-    <script src="script.js"></script>
+    <link rel="stylesheet" href="style.css" />
     <!--Import Google Icon Font-->
-    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+    <link
+      href="https://fonts.googleapis.com/icon?family=Material+Icons"
+      rel="stylesheet"
+    />
     <!--Import materialize.css-->
-    <link type="text/css" rel="stylesheet" href="css/materialize.min.css"  media="screen,projection"/>
+    <link
+      type="text/css"
+      rel="stylesheet"
+      href="css/materialize.min.css"
+      media="screen,projection"
+    />
 
     <!--Let browser know website is optimized for mobile-->
-    <meta name="viewport" content="width=device-width, initial-scale=1.0"/>
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
   </head>
   <body>
-     <h3>Create Study Set</h3>
-     <!--JavaScript at end of body for optimized loading-->
-     <script type="text/javascript" src="js/materialize.min.js"></script>
+    <nav class="nav-wrapper">
+      <a href="#" class="brand-logo">Studentia</a>
+      <a href="#" class="sidenav-trigger" data-target="slide_out"
+        ><i class="material-icons">menu</i></a
+      >
+      <ul id="nav-mobile" class="right hide-on-med-and-down">
+        <li><a href="findStudySet.html">Search</a></li>
+        <li><a href="index.html">Home</a></li>
+      </ul>
+    </nav>
+
+    <div class="row">
+      <div class="col s4 left-align teal lighten-4 z-depth-3 create-box">
+        <p class="flow-text white-text">First Step to Create Study Set</p>
+
+        <div class="input-field col s12">
+          <input type="text" id="user_id" name="user_id" required/>
+          <label for="user_id">User ID:</label>
+        </div>
+
+        <div class="input-field col s12">
+          <input type="text" id="title" name="title" required/>
+          <label for="title">title:</label>
+        </div>
+
+        <div class="input-field col s12">
+          <input type="text" id="subject" name="subject" required/>
+          <label for="subject">Subject:</label>
+        </div>
+
+        <div class="input-field col s12">
+          <input type="text" id="description" name="description" required/>
+          <label for="description">Description:</label>
+        </div>
+
+        <div class="input-field col s12">
+          <input type="text" id="university_id" name="university_id" required/>
+          <label for="university_id">University ID:</label>
```

Comment:
To confirm, this will be changed to a dropdown in a later PR?

Reply:
Yes